src/modules/auth/components/Input.tsx

```diff
@@ -0,0 +1,14 @@
+type InputProps = {
+  type: string;
+  placeholder: string;
+};
+
+export const Input = ({ type, placeholder }: InputProps) => {
+  return (
+    <input
+      type={type}
+      placeholder={placeholder}
+      className="input input-bordered my-2 h-10 w-full text-sm"
```

Comment:
Uvek treba izbegavati dodavanje margina unutar komponente koje uticu na "spoljne" elemente. Jer sta ako hocemo da stavimo 2 input jedan pored drugog sa manjom marginom... Moracemo da override-ujemo ovaj sitl, a to nece da nam bude zanimljivo...
Da li nam je potrebna class-a `h-10` na ovom elementu pored `input input-borderd`? 
Takodje bih izbacio `text-sm`, da bi se sto vise oslanjali na `dasiyui`, zbog tema koje ima...